project-base/src/DataFixtures/Demo/SettingValueDataFixture.php

```diff
@@ -122,6 +122,11 @@ public function load(ObjectManager $manager)
                 t('| Demo eshop', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale),
                 $domainId
             );
+            $this->setting->setForDomain(
+                SeoSettingFacade::SEO_ROBOTS_CONTENT,
+                'Disallow: /admin',
+                $domainId
+            );
```

Comment:
Is this change supposed to be in robots.txt in production or is it intended as interesting demo data?

Reply:
It's just for demo data purposes.